plugins/jira/jira.plugin.zsh

```diff
@@ -59,6 +59,16 @@ open_jira_issue () {
   fi
 }
 
+jira_url_help() {
+  cat << EOF
+JIRA url is not specified anywhere.
+Valid options, in order of preference:
+  .jira-url file
+  $HOME/.jira-url file
```

Comment:
Would it be clearer to escape the `$HOME` so it shows up as a literal `$HOME` in the output, instead of the current user's particular path? For example, I get this when I run it.

➜  ~  jira
JIRA url is not specified anywhere.
Valid options, in order of preference:
  .jira-url file
  /Users/janke/.jira-url file
  JIRA_URL environment variable